services/database/waitlist.ts

```diff
@@ -0,0 +1,97 @@
+import prisma from "@database";
+import { Waitlist } from "@prisma/client";
+import { WaitlistInput } from "models/Waitlist";
+/**
+ * getWaitlistRecord takes the classId and parentId and returns the associated waitlist
+ * @param classId
+ * @param parentId
+ */
+async function getWaitlistRecord(input: WaitlistInput): Promise<Waitlist> {
+    const waitlistRecord = await prisma.waitlist.findUnique({
+        where: {
+            parentId_classId: {
```

Comment:
what does this key `parentId_classId` do?

Reply:
Good question! When using findUnique, it has to be a primary key for the table. The primary key in this case is (parentId, classId). In Prisma, they will create a field specifically for the @@unique specified of the model. See https://www.prisma.io/docs/reference/api-reference/prisma-client-reference/#findunique.